lib/dotcom_web/templates/trip_plan/_options.html.eex

```diff
@@ -1,12 +1,12 @@
-<h6>When</h6>
+<label>When</label>
 <%=
   render "_departure.html", Map.merge(assigns, %{
     title: format_plan_type_for_title(@query),
     prefix: "trip-plan-departure",
     datetime: datetime_from_query(@query)
   })
 %>
-<h6>Modes</h6>
+<label>Modes</label>
```

Comment:
**question**: Do `<label>` elements make sense here? This label has no association made with another element when used here like this.

Reply:
No elements really make sense there. But, the way things are rendered doesn't build a cohesive form. So, for now, unless we want to rewrite the whole thing, we have to choose some element to go there. I'm open to whatever.